pkg/config/agentmanagement.go

```diff
@@ -181,6 +191,21 @@ func (am *AgentManagementConfig) SleepTime() (time.Duration, error) {
 	return time.ParseDuration(am.PollingInterval)
 }
 
+// jitterTime returns a random duration in the range [minJitterDuration, maxJitterDuration).
+func jitterTime(minJitterDuration time.Duration, maxJitterDuration time.Duration) time.Duration {
+	// If either duration is negative, or the maximum duration is 0 return no jitter.
+	if minJitterDuration < 0 || maxJitterDuration <= 0 {
```

Comment:
If min and max are not going to be configurable I'm not sure what's the use case of having these checks here